src/utils.ts

```diff
@@ -193,7 +197,7 @@ const hrtimeBigint: () => bigint = typeof (globalThis as { process?: { hrtime?:
  */
 export const hrtimeNow = () => nToMs(Number(hrtimeBigint()))
 
-export const now = performance.now.bind(performance)
+export const performanceNow = performance.now.bind(performance)
```

Comment:
It's part of a documented public API. If you rename it, it must be flagged as a breaking change and the documentation must be updated accordingly.

Reply:
this is not what you get from our package. ;). Important is the index.ts exports.